brvs-core/src/main/java/iroha/validation/transactions/plugin/impl/sora/SoraDistributionPluggableLogic.java

```diff
@@ -333,7 +366,7 @@ private void sendDistributions(Map<String, List<Transaction>> distributionTransa
       Map<String, BigDecimal> toDistributeMap,
       BigDecimal fee,
```

Comment:
The whole method implementation is too complex. I hope we will refactor it  because now  it is difficult to maintain.